lms/djangoapps/lti_provider/users.py

```diff
@@ -53,9 +55,13 @@ def create_lti_user(lti_user_id, lti_consumer):
             edx_user = User.objects.create_user(
                 username=edx_user_id,
                 password=edx_password,
-                email='{}@lti.example.com'.format(edx_user_id)
+                email='{}@lti.example.com'.format(edx_user_id),
```

Comment:
Does it make sense to configure the e-mail domain? The e-mail addresses aren't going to be meaningful (they're only there because the User model requires them), so example.com seems like the appropriate domain. If we were to configure them as (say) @harvard.edu, it would just mean that there would be real-looking invalid e-mails in the system. This way, at least, we could replace the example.com addresses later depending on what way the user management strategy goes.